src/routes/transactions/entities/propose-transaction.dto.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { Operation } from '../../../domain/safe/entities/operation.entity';
+import { ProposeTransactionDto as DomainProposeTransactionDto } from '../../../domain/transactions/entities/propose-transaction.dto.entity';
+
+export class ProposeTransactionDto implements DomainProposeTransactionDto {
+  @ApiProperty()
+  to: string;
+  @ApiProperty()
+  value: string;
+  @ApiPropertyOptional({ type: String, nullable: true })
+  data: string | null;
+  @ApiProperty()
+  nonce: string;
+  @ApiProperty()
+  operation: Operation;
+  @ApiProperty()
+  safeTxGas: string;
+  @ApiProperty()
+  baseGas: string;
+  @ApiProperty()
+  gasPrice: string;
+  @ApiPropertyOptional({ type: String, nullable: true })
+  gasToken: string | null;
```

Comment:
Since `gasToken` is required, this signature also needs to be updated.

Reply:
Fixed in [0b60e2e](https://github.com/safe-global/safe-client-gateway-nest/pull/292/commits/0b60e2e077ddd4ba2cec258d3903a52f5141487d)